voskcli/transcribe.py

```diff
@@ -67,6 +75,21 @@ def model_path(model):
 
 
 def write_captions_paragraph(vtt, paragraph):
+    '''
+    Append time-coded paragraphs to vtt file.
+    
+    :param vtt: Empty or partially filled WebVTT file to write transcriptions to
+    :type vtt: WebVTT
+    :param paragraph: Transcription data segment containing 1-2 lines of up to 35 chars.
+        Entries for each word contain the following JSON like key/value pairs:
```

Comment:
“JSON like key/value pairs” probably meany a Python dictionary.
In Python, `{'a': 1}` is a dictionary.
It's very much like a `Map` in Java or a JSON object.

What I would find helpful is to know, that the structure is that this expects a list of lists, containing dictionaries, with the outer list representing the paragraph, the inner lists, representing the lines and the dictionaries representing the words:

```py
[
  [{'confidence': float, 'start': float, 'end': float, 'word': string}, …], # line 1
  … # line 2-n
]
```